FordLabs/retroquest
# Sort action items

## Overview
Allows the user to sort the action items by date created. This keeps it in feature parity with the other columns that sort by start count.

### Demo
![image](https://user-images.githubusercontent.com/6293337/44435619-c129bb00-a57e-11e8-962b-3870112ccdee.png)

## Testing Instructions
1. Add in some action items and change your system date so that the dates appear different.
2. Click on the sort button
3. You should see them sort
4. Uncheck the sort button and watch them go back to their original order.
